packages/liquidation-sdk-viem/examples/whitelistedMarkets.ts

```diff
@@ -82,33 +82,35 @@ export const check = async <
   const ethPriceUsd = safeParseNumber(wethPriceUsd, 18);
 
   return Promise.all(
-    (positions ?? []).map(async (position) => {
+    (positions ?? []).map(async ({ instance: position, type: ptype, ...d }) => {
       const user = position.user;
       const market = position.market;
-      const seizableCollateral =
-        position.seizableCollateral !== undefined &&
-        position.seizableCollateral !== 0n
-          ? { value: position.seizableCollateral, preLiquidation: false }
-          : { value: position.preSeizableCollateral, preLiquidation: true };
 
-      if (seizableCollateral.value === undefined)
+      const isPreLiquidation = ptype === "PreLiquidationPosition";
+      const seizableCollateral = position.seizableCollateral;
+
+      if (seizableCollateral === undefined) {
         return console.warn(`Unknown oracle price for market "${market.id}"`);
+      }
+      if (seizableCollateral === 0n) {
+        console.warn(
+          `Nothing seizable for ${position.marketId}:${position.user}. This should've been filtered out sooner.`,
+        );
+        return;
+      }
 
-      if (
-        seizableCollateral.preLiquidation &&
-        position.preLiquidation === undefined
-      )
+      if (isPreLiquidation && position.preLiquidation === undefined)
```

Comment:
```suggestion
      if (isPreLiquidation)
```

Can `position.preLiquidation` actually be undefined?